kDriveCore/Data/Models/DriveError.swift

```diff
@@ -235,7 +235,7 @@ public struct DriveError: Error, Equatable {
     private static let decoder = JSONDecoder()
 
     /// A specific, not user facing, not localized error
-    private var underlyingError: Error?
+    internal var underlyingError: Error?
```

Comment:
We don't need `internal` as this is the default access level

Reply:
I'm aware, I wanted to be explicit about it as we kind of order parameters given public / private access level. 

I'll remove it asap.